packages/frontend/src/pages/dashboard/[guildId]/modmail/settings/index.tsx

```diff
@@ -0,0 +1,126 @@
+import type { APIChannel, APIDMChannel, APIGroupDMChannel } from 'discord-api-types/v10';
+import { useMemo } from 'react';
+import ConfigForm from '~/components/Config/ConfigForm';
+import ConfigOption from '~/components/Config/ConfigOption';
+import { ConfigOptionCollection } from '~/components/Config/ConfigOptionCollection';
+import ConfigPageFrame from '~/components/Config/ConfigPageFrame';
+import { TextArea } from '~/components/Config/TextArea';
+import type { DropdownGroupedOptions } from '~/components/Dropdown';
+import Dropdown from '~/components/Dropdown';
+import PageMeta from '~/components/PageMeta';
+import * as Text from '~/components/Text';
+import useGuildInfo from '~/hooks/useGuildInfo';
+import useModmailSettings from '~/hooks/useModmailSettings';
+
+const allowedChannelTypes = [
+	0, // GUILD_TEXT
```

Comment:
You can import straight from `discord-api-types/v10`. Should work fine on the web, use the `ChannelTypes` enum.